.gitignore

```diff
@@ -78,3 +78,9 @@ plots/
 cmake-build-debug/
 clang-tidy-build/
 libbuild/
+
+
+# Data & config files
+
+.data/
+.env
```

Comment:
why is this needed?

Reply:
Just a quality-of-life improvement for developers like me who like to store data and environment variables in the repos directory without adding it to git vcs.